jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/impl/OracleSqlDialectIT.java

```diff
@@ -696,7 +696,7 @@ public void testTimestampLocalTZ() throws SQLException {
         executeUpdate("ALTER SESSION SET TIME_ZONE = 'UTC'");
         final String query = "SELECT C15 FROM %s";
         ResultSet resultJDBC = runQueryJDBC(query);
-        matchNextRow(resultJDBC, "30-APR-18 06.00.05.000000 PM");
+        matchNextRow(resultJDBC, "30-APR-18 07.00.05.000000 PM");
```

Comment:
Please make this more robust  against changes. E.g. by using regular expressions instead of hard-coded values.
This is tight coupling.

Reply:
I disagree, the JDBCAdapter does not exist in isolation, but only makes sense with the exasol database. If the exasol database changes this will affect our customers, and we should also be aware of that.